crates/issue/src/benchmarking.rs

```diff
@@ -35,6 +35,10 @@ fn dummy_public_key() -> BtcPublicKey {
     ])
 }
 
+const DUMMY_BTC_ADDRESS: BtcAddress = BtcAddress::P2PKH(H160([
```

Comment:
Can you make it a conditionally compiled function in BtcAddress, such that we can do `BtcAddress::dummy()`? That way we don't have to redefine it in every benchmark